### ISSUE TITLE
Adding HR to department

### ISSUE THREAD

**Issue by abprince** [sapplica/sentrifugo#126]
Not able to add HR to deparment

**Comment by 4poorv** [comment]
Faced and resolved a related Issue.
can you please elaborate what's the issue you are facing exactly? Maybe post screenshots
Please confirm that the employee does have **Role** as 'HR Manager'

**Comment by abprince** [comment]
Hi there
as per the guide and instruction I have created Business unit and Different
Departments , like HR, Admin, Production , IT and also I created HR manager
under HR department , and I able to enroll Leave management option and only
HR can apply can apply for leave ,rest of the department members cannot
apply for leave nor able to create Leave Management  Options please check
the attached screenshot

[image: Inline image 2]


[image: Inline image 3]

[image: Inline image 4]

[image: Inline image 5]

[image: Inline image 6]

**Comment by 4poorv** [comment]
The screenshots are not visible.
Also please check by placing HR manager into the same department as employee applying for leave.
Thanks

**Comment by abdoolnerc** [comment]
Hello,
When configuring 'Leave Management' must I assign a HR manager to every business unit and department?

**Comment by sundayayandele** [comment]
@abdoolnerc, yes you have to assign a HR manager while setting up the leave management

**Comment by arbabs** [comment]
Is there a way without creating HR manager in every Business Unit I can assign HR Manager from the HR Unit.

**Comment by akoshodi** [comment]
@arbabs  You will have to update the code to achieve this. The files to update are those related to leave management (leavemanagement model for the scenario you described)
